data-reconciliation-app/docs/adr/01-data-ingest.md

```diff
@@ -6,17 +6,17 @@ Approved
 
 ## Context
 
-We need to build an API that ingests data into our system. Once ingested, we need to store this data in the K-V store so we can reconcile and report out on the data.
+We need to build an API that ingests data into our system. Once ingested, we need to store this data in the K-V store so we can reconcile it and get reports on the ingested data.
 
 Gathered requirements:
 
 - Members of the network will decide on a data schema ahead of ingest.
-- Members and Users of the network can ingest data via the API. Data will be ingested via JSON which will include:
+- Members and users of the network can ingest data via the API. Data will be ingested via JSON, which will include:
```

Comment:
we can also ingest via csv is that correct?

Reply:
yes, have updated it in the ADR.